### PR TITLE
Added Main Menu for MultiPlayer and SinglePlayer

### PR DESCRIPTION
**Pull request by Husaria-Game**

### CODE REVIEW

File: Assets/Scripts/Cristopher's/ChooseFactionScript1.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
```
Comment (G-Captain) [inline Assets/Scripts/Cristopher's/ChooseFactionScript1.cs]: Please change naming to sth more understandable than "<scriptName>1" and "<scriptName>2"

File: Assets/Scripts/Cristopher's/ChooseFactionScript1.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    void Start()
+    {       
+        startGame.enabled = false;
+        isClicked = false;
+    }
+
+    void Update()
+    {
+            switch (isClicked)
```
Comment (G-Captain) [inline Assets/Scripts/Cristopher's/ChooseFactionScript1.cs]: please change switch case into if statement for boolean value

File: Assets/Scripts/Cristopher's/ChooseFactionScript2.cs [hunk 3]
```diff
@@ -0,0 +1,71 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
```
Comment (G-Captain) [inline Assets/Scripts/Cristopher's/ChooseFactionScript2.cs]: Same as above, ChooseFactionScript2 does not tell as much as ChooseSecondFaction, also you can skip adding "Script" in name

File: Assets/Scripts/Cristopher's/ChooseFactionScript2.cs [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    void Start()
+    {
+        startGame.enabled = false;
+        isClicked = false;
+    }
+
+    void Update()
+    {
+        switch (isClicked)
+        {
```
Comment (G-Captain) [inline Assets/Scripts/Cristopher's/ChooseFactionScript2.cs]: same as above, if else for boolean

File: Assets/Scripts/Cristopher's/ChooseFactionScriptWithComputer.cs [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    void Start()
+    {
+        startGame.enabled = false;
+        isClicked = false;
+    }
+
+    void Update()
+    {
+        switch (isClicked)
+        {
```
Comment (G-Captain) [inline Assets/Scripts/Cristopher's/ChooseFactionScriptWithComputer.cs]: same as above

File: Assets/Scripts/Cristopher's/MouseOverFaction1.cs [hunk 6]
```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+using System;
+
```
Comment (G-Captain) [inline Assets/Scripts/Cristopher's/MouseOverFaction1.cs]: same as before

File: Assets/Scripts/Cristopher's/MouseOverFaction2.cs [hunk 7]
```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+using System;
+
```
Comment (G-Captain) [inline Assets/Scripts/Cristopher's/MouseOverFaction2.cs]: as before

File: Assets/Scripts/Cristopher's/ChooseFactionScript1.cs (hunk 2 above)
Comment (Husaria-Game) [reply Assets/Scripts/Cristopher's/ChooseFactionScript1.cs]: Done!

File: Assets/Scripts/Cristopher's/ChooseFactionScript2.cs (hunk 3 above)
Comment (Husaria-Game) [reply Assets/Scripts/Cristopher's/ChooseFactionScript2.cs]: Done!

File: Assets/Scripts/Cristopher's/ChooseFactionScript2.cs (hunk 4 above)
Comment (Husaria-Game) [reply Assets/Scripts/Cristopher's/ChooseFactionScript2.cs]: Done!

File: Assets/Scripts/Cristopher's/ChooseFactionScriptWithComputer.cs (hunk 5 above)
Comment (Husaria-Game) [reply Assets/Scripts/Cristopher's/ChooseFactionScriptWithComputer.cs]: Done!

File: Assets/Scripts/Cristopher's/MouseOverFaction1.cs (hunk 6 above)
Comment (Husaria-Game) [reply Assets/Scripts/Cristopher's/MouseOverFaction1.cs]: Done!

File: Assets/Scripts/Cristopher's/MouseOverFaction2.cs (hunk 7 above)
Comment (Husaria-Game) [reply Assets/Scripts/Cristopher's/MouseOverFaction2.cs]: Done!

**Comment by Husaria-Game** [comment]
Asking again for merging work

**Pull request merged**